docs/clients/arming_handler.rst

```diff
@@ -0,0 +1,149 @@
++--------+-------------------------------------+-----------------+-----------+-----------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|| 10    || disarm_song_option                 || get, set, save || uint8    ||          || The state that determines if an dhow many times the module will play its disarm song after coming to a stop. See row below for each state:                                                                                            |
+||       ||                                    ||                ||          ||          || 0 = Never Play: The disarm song is skipped entirely, and the module will transition directly to its final state after stopping.                                                                                                       |
+||       ||                                    ||                ||          ||          || 1 = Play Once: The disarm song will play once, and then the module will transition to its final state.                                                                                                                                |
+||       ||                                    ||                ||          ||          || 2 = Play Continuously: The disarm song will play continuously until the module is armed again or commanded to sping without arming through IQUART. The song will never finish, so the module will never transtion to its final state. |
++--------+-------------------------------------+-----------------+-----------+-----------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|| 11    || manual_arming_throttle_source      || get, set, save || uint8    ||          || The throttle source used as its armed throttle source for manual arming. The default manual throttle source is Unknown. In order to use manual arming, a throttle source must be set first. See row below for each state:             |
+||       ||                                    ||                ||          ||          || 0 = Unknown:                                                                                                                                                                                                                          |
+||       ||                                    ||                ||          ||          || 1 = Hobby: REFER TO Throttle Sources page when merged in docs                                                                                                                                                                         |
```

Comment:
Instead of REFER TO, can you make this a real internal reference link? I can show you how to do that if you are unfamiliar. Also this says "when merged in docs", which we should lose now